frontend/src/metabase/query_builder/components/DataSelector.jsx

```diff
@@ -286,18 +310,26 @@ export default class DataSelector extends Component {
                     }))
             }];
             return (
-                <AccordianList
-                    id="TablePicker"
-                    key="tablePicker"
-                    className="text-brand"
-                    sections={sections}
-                    searchable={true}
-                    onChange={this.onChangeTable}
-                    itemIsSelected={(item) => item.table ? item.table.id === this.getTableId() : false}
-                    itemIsClickable={(item) => item.table && !item.disabled}
-                    renderItemIcon={(item) => item.table ? <Icon name="table2" size={18} /> : null}
-                    hideSingleSectionTitle={true}
-                />
+                <div style={{ width: 300 }}>
+                    <AccordianList
+                        id="TablePicker"
+                        key="tablePicker"
+                        className="text-brand"
+                        sections={sections}
+                        searchable={true}
+                        onChange={this.onChangeTable}
+                        itemIsSelected={(item) => item.table ? item.table.id === this.getTableId() : false}
+                        itemIsClickable={(item) => item.table && !item.disabled}
+                        renderItemIcon={(item) => item.table ? <Icon name="table2" size={18} /> : null}
+                        hideSingleSectionTitle={true}
+                    />
+                    { isSavedQuestionList && (
+                        <div className="bg-slate-extra-light p2 text-centered">
+                            Is a question missing?
```

Comment:
There could be an integration test for this scenario! It's easy to test as there shouldn't be saved questions in the test database (or that's how it should be, I think that in metabase-lib branch we are not very careful with cleaning those up but that is easy fix).

I could help with adding tests early next week – QB can be a little tricky when testing with Enzyme.